opencti-platform/opencti-graphql/src/manager/telemetryManager.ts

```diff
@@ -113,6 +113,12 @@ export const addDisseminationCount = () => {
   }
 };
 
+export const addNLQQueryCount = () => {
+  if (filigranTelemetryMeterManager) {
+    filigranTelemetryMeterManager.addNLQQueryCount();
+  }
+};
```

Comment:
This will required some changes after https://github.com/OpenCTI-Platform/opencti/pull/10259 is merged